src/fontra/client/core/glyph-controller.js

```diff
@@ -619,6 +620,35 @@ export class StaticGlyphController {
   get flattenedPathHitTester() {
     return getRepresentation(this, "flattenedPathHitTester");
   }
+
+  getSelectionBounds(selection) {
+    let { point: pointIndices, component: componentIndices } =
+      parseSelection(selection);
+
+    pointIndices = pointIndices || [];
+    componentIndices = componentIndices || [];
+
+    const selectionRects = [];
+    if (pointIndices.length) {
+      const path = filterPathByPointIndices(this.instance.path, pointIndices);
+      if (path) {
+        selectionRects.push(path.getBounds());
+      }
+    }
+
+    for (const componentIndex of componentIndices) {
+      const component = this.components[componentIndex];
+      if (!component || !component.bounds) {
+        continue;
+      }
+      selectionRects.push(component.bounds);
+    }
+
+    if (selectionRects.length) {
```

Comment:
You can unconditionally do
```js
    return unionRect(...selectionRects);
```
As unionRect can handle an empty list.

Reply:
done.

Reply:
With "unconditionally" I meant you can leave out the "if" statement. There's no need for the .length check.

Reply:
Sorry. Changed it.